### PR TITLE
feat(nft-base-api): get wallet's assets endpoint

### PR DESCRIPTION
**Pull request by amacar**
### What changes are being made?
Added an endpoint that allows fetching all of the assets that the wallet has.

### Why are these changes necessary?
To easily fetch all of the user's assets on the frontend part of the application.

## Checklist

- [ ] Documentation _(if necessary)_
- [x] Tests _(if necessary)_
- [x] Ready to be merged

### CODE REVIEW

File: packages/nft-base-api/src/controllers/assets.ts [hunk 1]
```diff
@@ -61,6 +62,26 @@ export class AssetsController extends BaseController {
 		return this.respondWithBlockResource(transaction, request.query.transform, AssetResource);
 	}
 
+	public async showWalletAssets(request: Hapi.Request, h: Hapi.ResponseToolkit) {
```
Comment (KovacZan) [inline packages/nft-base-api/src/controllers/assets.ts:65]: Return `Promise<any>` so we don't violate `@typescript-eslint/explicit-module-boundary-types` rule

**Review by KovacZan — COMMENTED** [review]
Otherwise looks good :)

File: packages/nft-base-api/src/routes/assets.ts [hunk 2]
```diff
@@ -56,6 +56,19 @@ export const register = (server: Hapi.Server): void => {
 		},
 	});
 
+	server.route({
```
Comment (KovacZan) [inline packages/nft-base-api/src/routes/assets.ts:59]: Maybe pagination? 
```ts
 plugins: {
     pagination: {
        enabled: true,
    },
 },
```

Comment (amacar) [reply packages/nft-base-api/src/routes/assets.ts:59]: Will add

**Review by KovacZan — APPROVED** [review]

**Pull request merged**